boards/nucleo_h745zi_q.json

```diff
@@ -0,0 +1,43 @@
+{
+  "build": {
+    "core": "stm32",
+    "cpu": "cortex-m7",
+    "extra_flags": "-DSTM32H7xx -DSTM32H745xx",
+    "f_cpu": "480000000L",
+    "mcu": "stm32h745zit6",
+    "variant": "NUCLEO_H745ZI_Q"
```

Comment:
What is this field for?

Reply:
Which one of them do you mean: {`extra_flags`, `f_cpu`, `mcu`, `variant`} ?

Reply:
I meant `variant` field.

Reply:
It was just copy-paste from `nucleo-h743zi` board already supported.

Reply:
Changed to:
```
   "zephyr": {
      "variant": "nucleo_h745zi_q_m7"
    }
```